src/detect.h

```diff
@@ -250,6 +250,8 @@ typedef struct DetectPort_ {
 /** Info for Source and Target identification */
 #define SIG_FLAG_DEST_IS_TARGET         BIT_U32(26)
 
+#define SIG_FLAG_FILEDATA               BIT_U32(27) /**< signature has filedata keyword */
```

Comment:
Like I said in the last PR, I would like this to be a INIT flag (for Signature::init_data::flags) as we only use it during init. I want to preserve the runtime signature flag space.

Reply:
Ah. I was astonished indeed that signature flag space seemed to cheap.
I did not know about this `init_flags`